### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by Taras2552**
- [DEMO LINK](https://Taras2552.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Taras2552.github.io/layout_product-cards/report/html_report/)

- [ ] there 2 BEM blocks `card` and `stars` each in it own file
- [ ] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [ ] SCSS Variables are used for main values and placed in a **separate file**
- [ ] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [ ] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/index.html [hunk 1]
```diff
@@ -2,12 +2,43 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="grade__reviews">
+          <p class="grade__text">Reviews: 5</p>
+        </div>
+      </div>
+
+      <div class="price">
```
Comment (anastasiia-tilikina) [inline src/index.html:35]: price is not separate BEM block - it's an element of card. use proper class

File: src/index.html [hunk 2]
```diff
@@ -2,12 +2,43 @@
...
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac_images"
+        class="card__img">
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__id">Product code: 195434</p>
+
+      <div class="grade">
```
Comment (anastasiia-tilikina) [inline src/index.html:21]: same here

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
please make sure that you've fixed all comments

**Review by maxim2310 — CHANGES_REQUESTED** [review]
check you demo link and request review again
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Comment by Taras2552** [comment]
> Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Pull request closed without merging**